### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by atlehang66**
Here is my initial commit, it includes a full HTML structure, style as well as JavaScript for basic form controls.

### CODE REVIEW

File: js.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+// Getting references to HTML elements
+const addButton = document.getElementById('addButton');
```
Comment (Xolanisyav04) [inline js.js:2]: You need to use node.js to handle the backend for creating,reading, updating, and deleting of tasks, I think you need to refactor your code, NB: use node.js.

**Review by Xolanisyav04 — CHANGES_REQUESTED** [review]

File: js.js (hunk 1 above)
Comment (atlehang66) [reply js.js:2]: Okay will do. Thank you for your review!

**Review by Xolanisyav04 — APPROVED** [review]
Well done you did a good job! your To Do List is unique, you can merge.

**Review by Xolanisyav04 — APPROVED** [review]
This one is also fine I see you no longer have the clock

**Pull request merged**